testData/solutions/win32.json5

```diff
@@ -32614,7 +32614,43 @@
                         }
                     }
                 },
-                "inverseCapabilitiesTransformations": {}
+                "inverseCapabilitiesTransformations": {
```

Comment:
It looks like there isn't a high contrast inverse transformation. From what I recall you had some issues with that one. I think you can do the inverse transform by mapping `"default"` to `"http://registry\\.gpii\\.net/common/highContrast/enabled"` with value set to `false`. For rest you'd follow the theme mapping e.g. `"bw"` maps to `"http://registry\\.gpii\\.net/common/highContrastTheme"` with the value set to `"black-white"`. If you can, you could also set `"http://registry\\.gpii\\.net/common/highContrast/enabled"` with value set to `true`, however looking at the capability transform, it will take a theme value or the enabled preference set to mean there is a defined contrast.

Reply:
Yes, I had an inverse transformation in a [previous commit](https://github.com/klown/universal/commit/021236b30ef7f3d2e190809782a304db068c6300#diff-cb7e20a6c5ff97630ea4518024a845b2), see lines 32580 (`enabled`) and 32595 (`contrastTheme`).  It worked.  However, when running the solutions registry unit tests, the inverse for `contrastTheme` was found invalid with the message:

`ERROR: The capabilities transforms for win32.json5 -> net.gpii.uioPlus -> contrastTheme may result in data corruption.`

I'll compare your suggestions with what I did.

Reply:
@klown I don't know what that error message means, so if you're still getting that we may need to check with @amb26 or maybe @the-t-in-rtf.

Reply:
@jobara, I managed to get past the validation error by having the inverse transformation for `http://registry\\.gpii\\.net/common/highContrastTheme` by outputting a value of `regular-contrast` when `contrastTheme` is anything but `default`.  That makes sense because the inbound transformation for `contrastTheme` checks only a value of `regular-contrast` and does the mapping only if it isn't `regular-contrast`.  I'll push that soon.

Still wondering where in the inverse to output a value for `http://registry\\.gpii\\.net/common/highContrast/enabled`.

Reply:
Note that this is remarkably similar to what @sgithens did for desktop contrast in https://github.com/GPII/universal/pull/874

Reply:
> Note that this is remarkably similar to what @sgithens did for desktop contrast in #874

Yes, it looks like whomever wrote the `capabilitiesTransformations` transform of `highContrast` and `highContrastTheme` for the `com.microsoft.windows.highContrast` solution used the same rules for the `net.gpii.uioPlus` solution.  There is even a comment to that effect:
`// Use the same condition for applying a high contrast theme to windows`

I took the `capabilitiesTransformations` for UIO+ as given and worked on providing the missing `inverseCapabilitiesTransformations`.

Seeing as @sgithens modifications have been merged into master, I will bring the logic of the contrast transformation here into line with that, and, as appropriate, the inverse transformation.

However, I just noticed that UIO+ does not send any kind of "enabled" setting in its request with respect to high contrast, only a `contrastTheme` setting.  See the [example clip](https://www.irccloud.com/pastebin/dIHxYuY5/) from @jobara.  That suggests a simplification of the transformation rules here.  Or, perhaps, that UIO+ should be sending a `highContrast: true/false/default` setting.

Reply:
Okay, I've modified the `net.gpii.uioPlus` transformation for high contrast similar to what @sgithens has done for `com.microsoft.windows.highContrast`, but I've left the inverse transformation as is.  The inverse for the windows solution is a straight forward mapping of a windows value to the `enabled` common term.  The inverse for UIOPlus is from its contrast theme setting (e.g., `by`) to both the `enabled` common term and the common term for the theme (e.g., `black-yellow`).

Let me know if this is correct.